pyHS100/smartbulb.py

```diff
@@ -155,6 +177,10 @@ def color_temp(self, temp: int) -> None:
         if not self.is_variable_color_temp:
             return None
 
+        if temp < self.valid_temperature_range[0] or \
+                temp > self.valid_temperature_range[1]:
+            raise ValueError
```

Comment:
Please add an error message indicating the that the valid range was exceeded and add a tests for both error cases (too small and too large temperature), then it's good to go! :-)

Reply:
Done !